addons/calendar/models/calendar_event.py

```diff
@@ -1613,6 +1633,38 @@ def ics_datetime(idate, allday=False):
 
         return result
 
+    @api.model
+    def _get_contact_details_description(self, organizer, partners):
+        """Build sanitized HTML with the organizer details and the details
+        of the contact partner (the first partner which is not the organizer).
+        """
+        odoobot = self.env.ref('base.user_root')
+        contact_description = []
+        # Organizer
+        if organizer and organizer.id != odoobot.id:
```

Comment:
not sure if it's worth it or a great idea but perhaps in the case where the organizer is odoobot we could add the company info as the organizer ?